packages/console/src/pages/ApplicationDetails/ApplicationDetailsContent/ProtectedAppSettings/index.tsx

```diff
@@ -197,6 +199,24 @@ function ProtectedAppSettings({ data }: Props) {
             ))}
           </div>
         </FormField>
+        <FormField title="application_details.implement_jwt_verification">
+          <InlineNotification severity="alert">
+            <Trans
+              components={{
+                a: (
+                  <TextLink
+                    href={getDocumentationUrl(
+                      '/docs/recipes/protect-your-api/#validate-the-api-requests-authorization-token'
```

Comment:
Add a comment, we'll change the link later.

Reply:
ok